resources/lang/de/Config.properties

```diff
@@ -12,6 +12,9 @@ filtersStylingFilterPerRow1=1
 filtersStylingFilterPerRow2=2
 filtersStylingFilterPerRow3=3
 filtersStylingFilterPerRow4=4
+filtersShowSelectedFiltersCountLabel=Show selected filters count
+filtersShowSelectedFiltersDisabled=No
+filtersShowSelectedFiltersEnabled=Yes
```

Comment:
That's not German.

Reply:
I don't have the German translations. If you add them here, I'll update these.

Reply:
Ah, sorry, they were in the story after all. Missed them at first.